prisma/migrations/20231013203903_user_onboarding_step/migration.sql

```diff
@@ -0,0 +1,13 @@
+BEGIN;
+CREATE TYPE "OnboardingStep" AS ENUM ('Moderation', 'Buzz');
+ALTER TABLE "User" ADD COLUMN "onboardingSteps" "OnboardingStep"[];
+COMMIT;
+
+BEGIN;
+UPDATE "User" SET "onboardingSteps" = CASE
+  WHEN onboarded = false THEN ARRAY['Moderation', 'Buzz']::"OnboardingStep"[]
+  WHEN onboarded = true THEN ARRAY['Buzz']::"OnboardingStep"[]
+END;
+COMMIT;
+
+ALTER TABLE "User" DROP COLUMN "onboarded";
```

Comment:
I think you can set this all within the same transaction (BEGIN COMMIT) just to avoid doing it if something above errors out.